apps/settings/js/panels/root/README.md

```diff
@@ -0,0 +1,7 @@
+# Root
+
+## Trouble Shooting
+
+* If BT or NFC can not be enabled
+
+If BT or NFC can not enabled, please update **full image** instead of only update gecko or gaia. Since BT, NFC has its correspondent daemon in gonk layer. Gonk and Gecko should be paired if there's any modification.
```

Comment:
s/can not enabled/can not be enabled
s/update gecko or gaia/updating gecko or gaia
s/Since BT, NFC has its correspondent daemon/Since BT and NFC have their correspondent daemons